code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -113,7 +113,7 @@
 
 /mob/living/carbon/human/proc/check_block()
 	if(mind)
-		if(mind.martial_art && prob(mind.martial_art.block_chance) && in_throw_mode && !stat && !IsKnockdown() && !IsStun())
+		if(mind.martial_art && prob(mind.martial_art.block_chance) && mind.martial_art.can_use(src) && in_throw_mode && !stat && !IsKnockdown() && !IsStun())

```

Comment:
wait, looking at this, does this not bother to check if someone's restrained?

Reply:
Should I add it?